spec/requests/api/v1/targets/show_spec.rb

```diff
@@ -38,4 +39,24 @@
       expect(response).to have_http_status(:not_found)
     end
   end
+
+  context 'when the user is not authenticated' do
+    subject { post api_v1_targets_url, params: target_params, as: :json }
+    let(:target_params) do
+      {
+        target: attributes_for(:target).merge(topic_id: topic.id)
+      }
+    end
+
+    it 'returns the error messages as a json' do
+      subject
+      error_msg = I18n.t('devise.failure.unauthenticated')
```

Comment:
As the message is not so long, you can avoid using a variable, and just call `I18n.t('devise.failure.unauthenticated')` inside the expect.